extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php

```diff
@@ -28,6 +28,7 @@ class UserIdentityBox {
 
 	public $optionsArray = array(
 		'location',
+		'bio',
```

Comment:
`bio` is used by apps for their profiles. By adding this here, it's included in the attributes to be blanked out by `resetUserProfile`.